modules/auxiliary/dos/scada/siemens_siprotec4.rb

```diff
@@ -0,0 +1,36 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Auxiliary
+    include Msf::Exploit::Remote::Udp
```

Comment:
According to the metasploit style-tips documentation, use Two-space  :-). You may also want to checkout landed PR of other contributors so you can see what kind of styling formats you can follow.
https://github.com/rapid7/metasploit-framework/wiki/Style-Tips

Reply:
Also thanks for the document, I've changed the style.